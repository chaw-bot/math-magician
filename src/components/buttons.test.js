import React from 'react';
import renderer from 'react-test-renderer';
import Buttons from './buttons';

it('Render buttons', () => {
  const text = 'test';
  const calcObj = { string: 'test' };
  const onClick = jest.fn(() => {});
  const tree = renderer
    .create(<Buttons text={text} onClick={onClick} Obj={calcObj} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
