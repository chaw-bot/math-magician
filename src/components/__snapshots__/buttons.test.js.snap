// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render buttons 1`] = `
<button
  className="Calc-btn"
  onClick={[Function]}
  type="button"
>
  test
</button>
`;
